src/kernel/filesystem/vfs.zig

```diff
@@ -87,13 +87,13 @@ pub const FileSystem = struct {
     const Self = @This();
 
     ///
-    /// Close an open file, performing any last operations required to save data etc.
+    /// Close an open file or directory, performing any last operations required to save data etc.
```

Comment:
Maybe change this to say "an open node" since any node, not just directories and files, can be closed with these changes.